rskj-core/src/main/java/org/ethereum/vm/program/Program.java

```diff
@@ -1573,6 +1574,10 @@ public void callToPrecompiledAddress(MessageCall msg, PrecompiledContract contra
 
             byte[] out = contract.execute(data);
 
+            if (getBlockchainConfig().isRfs90()) {
+                this.returnDataBuffer = out;
```

Comment:
Is this all that's needed to implement `returnDataSize` and `returnDataCopy`?

Reply:
Yes, that's all. I'll add another test for checking `returnDataCopy` explicitly b/c atm I'm just checking `returnDataSize`